COA-857: Update Free Apps Archival documentation (#4382)
mendix/docs: content/en/docs/developerportal/deploy/mendix-cloud-deploy/_index.md

#### 1.1.1 Free Apps Archival{#free-apps-archival}

> If a Free App has been in Sleep Mode for three months or longer, the Technical Contact may be informed that the Free App may be archived in another two weeks. The project and model will be retained, but the running app, database, and files for the Free App will be deleted.

Reviewer: Hi @jeohist 
So you want to say that the app _may_ be archived and if it is going to be archived the Technical Contact _will_ be notified two weeks before?
In that case, I would rewrite it to say something like:
```suggestion
If a Free App has been in Sleep Mode for three months or longer, it may be archived. For Free Apps which are going to be archived, the Technical Contact will be informed two weeks beforehand. The project and model will be retained, but the running app, database, and files for the Free App will be deleted.
```

Author: Yep! I've committed your changes.
